Makefile

```diff
@@ -194,7 +194,7 @@ cmd/promtail/promtail-debug: $(APP_GO_FILES) pkg/promtail/server/ui/assets_vfsda
 # concurrency is limited to 4 to prevent CircleCI from OOMing. Sorry
 GOX = gox $(GO_FLAGS) -parallel=4 -output="dist/{{.Dir}}-{{.OS}}-{{.Arch}}" -arch="amd64 arm64 arm" -os="linux"
 dist: clean
-	CGO_ENABLED=0 $(GOX) ./cmd/loki
+	CGO_ENABLED=0 $(GOX) -osarch="windows/amd64" ./cmd/loki
```

Comment:
does `darwin/amd64` etc need to be included like for promtail and logcli below?

Reply:
I don't believe anyone needs the loki server on osx...

Reply:
Fair enough, but what was the default before? `linux/amd64`? Are we losing that here by specifying Windows?

Sorry, I don't remember how all of this was set up

Reply:
no, we specify `linux` as the os, multiple arches, they are matrix expanded. Furthermore we supply now `windows/amd64` specifically as well, which results in the usual linux builds, plus that one windows build

Reply:
cool, thanks for re-explaining